src/main/java/frc/robot/Constants.java

```diff
@@ -329,17 +329,17 @@ public static final class CameraConstants {
     public static final boolean kEnable = false;
     public static final List<CameraConfig> kCameraConfigs = List.of(
       new CameraConfig(
-        "camera1",
+        "reefCamera",
         new Transform3d(
-          0.196, -0.238, 0.26,
-          new Rotation3d(0.0, Units.degreesToRadians(-21.0), 0.0)),
+          0.279292, 0.137998, 0.317346,
+          new Rotation3d(Units.degreesToRadians(22.5), 0.0, Units.degreesToRadians(9.5))),
```

Comment:
`Rotation3d(roll, pitch, yaw)`. We have pitch  for the coral station camera (upward = negative), and yaw (counterclockwise = positive) for both cameras. The camera names appear to be reversed, and the `roll=22.5` should be `pitch=-22.5`.